kbi/0008/index.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   triple: git-annex; uninit; remove dataset annex
+
+KBI0008: Remove a dataset's annex
+=================================
+
+:authors: Christian Mönch <christian.moench@web.de>
+:discussion: https://github.com/datalad-handbook/book/issues/939
+:keywords: annex, remove annex, annex uninit
+
+This knowledge base item describes how to remove the annex from a DataLad
+dataset and propagate that change to siblings
+
+DataLad version used: this knowledge base item was created based on DataLad version 0.18.3
+
+
+The scenario
+------------
+
+You have a DataLad dataset that has annexed files and you decide that you
+don't want to use git-annex anymore in this dataset. This change should
+also be propagated to all siblings of the dataset.
+
+
+A few words of caution
+......................
+
+1. You should not remove the annex if you have large files, i.e. 1GB or larger, in your dataset. ``git`` is not intended for large files, which is the reason for the creation of ``git annex`` in the first place.
+
+2. If you remove the annex, your files will be untracked in your dataset directory. That means, they are not versioned at this moment. If you don't follow the instructions in this knowledge base item precisely you might lose those files. Please be careful when you remove the annex! Work on a copy, or make sure that you have a sibling that contains your data. We do not want you to lose data.
+
+
+Removing the annex
+------------------
+
+Go to the dataset from which you want to remove the annex and save the current state. Fetch all annexed data
+
+.. code-block:: console
+
+    > cd my_dataset
+    > datalad save
+
+Fetch all annexed data. Data that is not locally present when you remove the annex will not be locally present after removal of the annex.
```

Comment:
```suggestion
Fetch all annexed data. File contents that are not locally present when the annex is removed can no longer be retrieved via ``datalad get``.
```

Reply:
Irritatingly, the link destination will show up as file content. So there is content, but it is not the content of the file. That is an additional problem to not being able to ``datalad get`` content.